### PR TITLE
docs(grid): Updated docs for filtering with latest changes #1355

### PR DESCRIPTION
**Pull request by sboykova**
Related to IgniteUI/igniteui-angular#1355

### CODE REVIEW

File: en/components/grid_filtering.md [hunk 1]
```diff
@@ -73,14 +99,22 @@ this.grid.clearFilter();
 
 #### Initial filtered state
 
-It is possible to set the initial filtering state of the grid by an array of expressions to the `filteringExpressions` property of the grid.
+It is possible to set the initial filtering state of the grid by an array of expressions to the `filteringExpressionsTree` property of the grid.
```
Comment (bkulov) [inline en/components/grid_filtering.md]: You could remove the `initial`. The user could use this property to set grid's filtering state at any time.

Comment (bkulov) [reply en/components/grid_filtering.md]: Also this sentence doesn't correspond to the sample bellow. The sentence says `filteringExpressionsTree` should contain an array of expressions which is not true. It should contain an array of `IFilteringExpressionsTree` for each column the user wants to filter.

**Comment by yoshimis** [comment]
Please update codes in grid_filtering for JP.

**Comment by kdinev** [comment]
@sboykova Please apply the code updates to the JP version.

**Pull request merged**